### PR TITLE
Ported standard levels from https://mana.mozilla.org/wiki/display/SEC…

### PR DESCRIPTION
**Pull request by gdestuynder**
…URITY/Standard+documentation+levels

Initial import.

### CODE REVIEW

**Comment by gdestuynder** [comment]
Needs @marumari review
Deps: https://github.com/mozilla/wikimo_opsec/pull/24

File: Security/Standard_Levels.mediawiki [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+{| class="wikitable"
+|-
+! Risk Level
+| '''The importance of a risk as defined by its impact and probability.'''
+We express security risk levels qualitatively (low, medium, high, maximum).
+|-
+! Risk mitigation
+| Reduce the risk level through a technical or non-technical mean in order to:
+* Change it's impact.
```
Comment (marumari) [inline Security/Standard_Levels.mediawiki]: its  :)

File: Security/Standard_Levels.mediawiki [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+== Scoring levels ==
+These levels are used to score general effectiveness. Most commonly, these levels are used to score countermeasure effectiveness (controls) in security reports.
+
+Four levels are used to align with other security related levels, such as risk levels. Note these levels do not signify risk, and are more intended to provide a grade for a particular objective.
+
+{| class="wikitable"
+|-
+! Level
+! Color coding
```
Comment (marumari) [inline Security/Standard_Levels.mediawiki]: Is this column supposed to be here?

File: Security/Standard_Levels.mediawiki [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+{| class="wikitable"
+|-
+! Level
+! Coding rationale
+! Definition & expectations
+|-
+|-
+! <span style="padding: 0 32px;background-color:lightgrey;">OPTIONAL</span>
+|
+* See risk level's "low" rationale.
```
Comment (marumari) [inline Security/Standard_Levels.mediawiki]: Should this be a link instead?

File: Security/Standard_Levels.mediawiki [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+|
+* See risk level's "high" rationale.
+|
+* Should is very close to "must". It means something should be done unless there is a strong reason not to. Thus it's highly important.
+|-
+! <span style="padding: 0 32px;background-color:#E77471;">MUST</span>
+|
+* See risk level's "maximum" rationale.
+|
+* This must be done - there is no exception.
```
Comment (marumari) [inline Security/Standard_Levels.mediawiki]: What I mean by colloquial is that I would probably say something like "Mandatory, without exception."

**Comment by gdestuynder** [comment]
@marumari whatcha think r+-?

**Comment by marumari** [comment]
I'll take a look on Monday.  :)

**Comment by marumari** [comment]
Looks good to me, r+.

**Comment by gdestuynder** [comment]
Live at https://wiki.mozilla.org/Security/Standard_Levels
thanks @marumari !

**Pull request merged**